include/flatbuffers/flatc_pch.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FLATBUFFERS_FLATC_PCH_H_
+#define FLATBUFFERS_FLATC_PCH_H_
+
+// code_generator.cpp
+// idl_parser.cpp
+#include <cmath>
+
+// code_generator.h
+//
+// cpp_generator.cpp
+// go_generator.cpp
+// idl_gen_go.cpp
+// swift_generator.cc
+// util.cpp
+#include <sstream>
+
+// idl_gen_dart.cpp
+// idl_gen_js_ts.cpp
+// python_generator.cc
+#include <cassert>
+
+// java_generator.h
+//
+// idl_gen_go.cpp
+// idl_gen_lobster.cpp
+// idl_gen_lua.cpp
+// idl_gen_php.cpp
+// idl_gen_python.cpp
+// idl_parser.cpp
+#include <string>
+
+// code_generator.h
+// go_generator.h
+// python_private_generator.h
+// swift_generator.h
+//
+// idl_gen_python.cpp
+// java_generator.cc
+// python_generator.cc
+#include <vector>
+
+// flatc.cpp
+// idl_parser.cpp
+#include <list>
+
+// idl_gen_cpp.cpp
+// idl_gen_js_ts.cpp
+// idl_gen_kotlin.cpp
+// idl_gen_lobster.cpp
+// idl_gen_lua.cpp
+// idl_gen_python.cpp
+// idl_gen_swift.cpp
+#include <unordered_set>
+
+// code_generator.h
+// java_generator.h
+//
+// cpp_generator.cpp
+// go_generator.cpp
+// java_generator.cc
+// python_generator.cc
+// swift_generator.cc
+#include <map>
+
+// idl_gen_js_ts.cpp
+#include <unordered_map>
+
+// java_generator.h
+// python_private_generator.h
+//
+// idl_gen_json_schema.cpp
+// java_generator.cc
+// python_generator.cc
+#include <iostream>
+
+// idl_gen_kotlin.cpp
+#include <functional>
+
+// idl_gen_python.cpp
+// python_generator.cc
+#include <set>
+
+// java_generator.cc
+#include <iterator>
+
+// idl_parser.cpp
+// java_generator.cc
+// python_generator.cc
+#include <algorithm>
+
+// python_generator.h
+//
+// idl_parser.cpp
+// java_generator.cc
+#include <utility>
+
+// code_generator.h
+// go_generator.h
+// swift_generator.h
+#include <memory>
+
+// python_generator.cc
+#include <tuple>
+
+// flatbuffers
+#include "flatbuffers/code_generators.h"
+#include "flatbuffers/flatbuffers.h"
+#include "flatbuffers/flexbuffers.h"
+#include "flatbuffers/idl.h"
+#include "flatbuffers/util.h"
+
+#endif // FLATBUFFERS_FLATC_PCH_H_
```

Comment:
can `flatc_pch.h` depend on `pch.h`, since it is largely an extension?

Reply:
So should i just remove all the comments? 
That would make sense if i include `pch` from `flatc_pch`